pkg/pkgmgmt/client/filesystem.go

```diff
@@ -45,6 +45,9 @@ type FileSystem struct {
 
 func (fs *FileSystem) List() ([]string, error) {
 	parentDir, err := fs.GetPackagesDir()
+	if err != nil {
+		return nil, errors.Wrap(err, "could not get package directory")
```

Comment:
Can you switch this to use fmt.Errorf and the "%w" placeholder instead of the pkg/errors package? We are trying to stop using the errors package (#2009).